# goal pose from current pose
sensor_msgs/JointState goal_pose

# duration in seconds
float32 duration